### ISSUE TITLE
OSC Commands freeze the camera - One X2

### ISSUE THREAD

**Issue by cristianoccazinsp** [Insta360Develop/Insta360_OSC#14]
The following commands cause the entire camera to freeze (it stops receiving commands and the screen does not respond to touches)

- `camera.delete` --> completely freezes the camera
- (not a command) `/osc/info` --> freezes the OSC api, and it stops accepting HTTP requests.

I'm using the Insta360 One X2, firmware version v1.0.41

**Comment by Tianweihaihaihai** [comment]
yes the camera will freeze if receive uncorrect command, pls check the command
